MadLibs

```diff
@@ -3,226 +3,85 @@ templ = random.randint(1, 3) #selecting 1 of 3 templates
-    Place = input("Please type a place")
-    Adjective2 = input("Please type another adjective")
-    while True:
-        if Adjective2 == Adjective:
-            print("Please do not repeat your answer.")
-            Adjective2 = input("Please type another adjective")
-        break
-    Magical_Creature = input("Please type a name of a Magical Creature in plural")
-    Adjective3 = input("Please type another adjective")
-    while True:
-        if Adjective3 == Adjective:
-            print("Please do not repeat your answer.")
-            Adjective3 = input("Please type another adjective")
-        if Adjective3 == Adjective2:
-            print("Please do not repeat your answer.")
-            Adjective3 = input("Please type another adjective")
-        break
-    Magical_Creature2 = input("Please type another name of a Magical Creature in plural")
-    while True:
-        if Magical_Creature2 == Magical_Creature:
-            print("Please do not repeat your answer.")
-            Magical_Creature2 = input("Please type another name of a Magical Creature in plural")
-        break
-    Noun = input("Please type a noun")
-    Noun2 = input("Please type another noun")
-    while True:
-        if Noun2 == Noun:
-            print("Please do not repeat your answer.")
-            Noun2 = input("Please type another noun")
-        break
-    Noun3 = input("Please type another noun")
-    while True:
-        if Noun3 == Noun:
-            print("Please do not repeat your answer.")
-            Noun3 = input("Please type another noun in plural")
-        if Noun3 == Noun2:
-            print("Please do not repeat your answer.")
-            Noun3 = input("Please type another noun in plural")
-        break
-    Adjective4 = input("Please type another adjective")
-    while True:
-        if Adjective4 == Adjective:
-            print("Please do not repeat your answer.")
-            Adjective4 = input("Please type another adjective")
-        if Adjective4 == Adjective2:
-            print("Please do not repeat your answer.")
-            Adjective4 = input("Please type another adjective")
-        if Adjective4 == Adjective3:
-            print("Please do not repeat your answer.")
-            Adjective4 = input("Please type another adjective")
-        break
-    Noun4 = input("Please type another noun")
-    while True:
-        if Noun4 == Noun:
-            print("Please do not repeat your answer.")
-            Noun4 = input("Please type another noun in plural")
-        if Noun4 == Noun2:
-            print("Please do not repeat your answer.")
-            Noun4 = input("Please type another noun in plural")
-        if Noun4 == Noun3:
-            print("Please do not repeat your answer.")
-            Noun4 = input("Please type another noun in plural")
-        break
-    Number = input("Please type a number")
-    Measure_of_time = input("Please type a measure of time")
-    Verb = input("Please type a verb ending in ing")
```

Comment:
you have no way of verifying if the users inputed verb ends with an -ing. Instead of asking that you could ask for a regular verb and in the text before format adding it after the input . (this would miss the irregular verbs with exceptions, but the text would make more sense) E.g. {}ing